chrome/content/zotero/HiddenBrowser.jsm

```diff
@@ -229,9 +234,41 @@ const HiddenBrowser = {
 	destroy(browser) {
 		var frame = browserFrameMap.get(browser);
 		if (frame) {
+			RemoteResourceBlockingObserver.unwatch(browser);
 			frame.destroy();
 			Zotero.debug("Deleted hidden browser");
 			browserFrameMap.delete(browser);
 		}
 	}
 };
+
+const RemoteResourceBlockingObserver = {
+	_observerAdded: false,
+	_browsingContextIDs: new Set(),
+	
+	observe(subject) {
+		let channel = subject.QueryInterface(Ci.nsIHttpChannel);
+		if (this._browsingContextIDs.has(channel.topBrowsingContextId)
```

Comment:
This changes to `browserId` in 115. We should add some code to try to support that.

https://searchfox.org/mozilla-central/rev/887d4b5da89a11920ed0fd96b7b7f066927a67db/netwerk/protocol/http/nsIHttpChannel.idl#467